### ISSUE TITLE
Debugger - Video RAM window is blank even when running project

### ISSUE THREAD

**Issue by cosmoddd** [godotengine/godot#68899]
___

***Bugsquad note:** This issue has been confirmed several times already. No need to confirm it further.*

___

### Godot version

4.0 beta 5

### System information

Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz   3.70 GHz  32.0 GB RAM  Nvidia Geforce RTX 2060

### Issue description

**What doesn't work:**
The entire Video Ram window remains empty during playmode.

**How do you expect it to work instead?**
There should be a list of files that are in use while the game is playing.

![2022-11-16 19_20_10-street scene - composite tscn - Tales From Off-Peak City RPG - Godot Engine](https://user-images.githubusercontent.com/9627628/202870329-9f53bfa1-8b61-416d-b946-24caeb263db6.png)

### Steps to reproduce

Simply play a project and open the **Video RAM** tab of the **Debugger window** to see that it's empty.

### Minimal reproduction project

N/A  (this has happened with every project I've tested)

**Comment by Calinou** [comment]
This is a duplicate of another issue, but I can't find it right now. The VRAM profiler isn't reimplemented yet as of 4.0.beta5.

**Comment by TokisanGames** [comment]
I also can't find an open duplicate for GD4. In Godot 3 I was pushing 2GB of vram. Now in GD4, I'm around 3.5gb. Where did it all come from!!? Looking forward to this as we move closer towards release and optimizations of the engine and my game.

**Comment by PLyczkowski** [comment]
I still have a blank list in 4.2 stable.

**Comment by lemilonkh** [comment]
This is still the case in 4.2.1.
Is anyone currently looking into this or should I give it a shot?

**Comment by akien-mga** [comment]
Feel free to give it a shot!

**Comment by szunami** [comment]
I ran into this issue with the Forward+ Renderer, but if I switch to the Compatibility Renderer, I do get data (I'm on 4.1.3). I realize this isn't a great solution, but it unblocked me.

**Comment by dqdthanhthanh** [comment]
Doesn't work on Mac air m1 with Godot 4.2.2 and Godot 4.3 dev5. Still works normally on Godot 3.

**Comment by clayjohn** [comment]
Fixed by https://github.com/godotengine/godot/pull/92000

This will be fixed in 4.3 beta 1 and later releases
